letsApply/Controllers/TabBarViewController/HomeScreenViewController/UICollectionViewCell/JobCategoryCell.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  JobCategoryCell.swift
+//  letsApply
+//
+//  Created by Reuben Simphiwe Kuse on 2024/12/17.
+//
+
+import UIKit
+
+class JobCategoryCell: UICollectionViewCell {
+    static let reuseidentifier = "JobCategoryCell"
```

Comment:
You con't have to put this here. I think there is an extension that allows you to access `cellID`. I will send it to you.